torch/csrc/jit/passes/quantization.cpp

```diff
@@ -835,6 +835,21 @@ void InsertQuantDeQuantHelper::run(
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
 
+  // We only need to register new parameters if the graph has
+  // been quantized before
+  if (values_to_qparams_.count(graph.get())) {
+    for (auto& pr : values_to_qparams_.at(graph.get())) {
+      Value* v = pr.first;
+      const std::unordered_map<std::string, IValue>& qparams = pr.second;
+      for (const auto& pr : qparams) {
+        const auto& name = pr.first;
+        const auto& qparam = pr.second;
+        module._ivalue()->setAttr(v->debugName() + name, qparam);
```

Comment:
Have you checked that it works for a module with several submodules of the same type? If m has conv1 and conv2, their qparams can be different, but this code will always set the same values to them for each instance of the conv module. I think there must be some calls to `get_qparams` somewhere here.

Reply:
Yeah I'm aware that it doesn't work, I plan to add the support in a separate PR

Reply:
In that case, we need to have different graphs for each type of qparams, since different qparams might correspond to different quantize calls

Reply:
Why would two convs have separate graphs? The graphs would be the same, and only the data (attributes) of the particular modules would differ. Could you please elaborate?

Reply:
I'm talking about quantized graph, input graph is the same for conv module, but if we have want to quantize one conv module instance using per tensor affine and the other using per channel affine, then we'll need to produce two different quantized graphs.

Reply:
What if I just want to quantize all convs the same way and I have 2+ of them. I was talking about this case, which seemed to me more frequent - in this case the graph would be the same and the current code will not give correct results.

Reply:
Oh, I see, yeah, `values_to_qparams` should be indexed by Module as well, I'll change this